lib/s3ff/view_helper.rb

```diff
@@ -32,24 +32,31 @@ def include_s3ff_templates
                 <button class="close" data-dismiss="alert" data-unique_id="{{:unique_id}}" data-target="#upload-{{:unique_id}}" style="margin-left:1em;" type="button">
                   <span aria-hidden="true">&times;</span><span class="sr-only">Remove</span>
                 </button>
-                <span class="fa fa-file-o"></span>
-                {^{:result.filename}}
+                <span class="filename">
+                  <span class="fa fa-file-o"></span>
+                  {^{:result.filename}}
+                </span>
               </div>
             </div>
           {^{else}}
-            {^{if placeholder}}
-              {^{if placeholder.url || placeholder.filename}}
-                <div class="thumbnail" id="upload-{{:unique_id}}">
-                  {^{if placeholder.url}}
-                    <img data-link="src{:placeholder.url}">
-                  {{/if}}
-                  {^{if placeholder.filename}}
-                    <div class="caption">
-                      <span class="fa fa-file-o"></span>
-                      {^{:placeholder.filename}}
-                    </div>
-                  {{/if}}
-                </div>
+            {^{if !progress_pct}}
```

Comment:
now that i look at it, is it cleaner to replace the entire `div.progress style="display:none;"` section above with this section?




just shift the entire chunk into the